users/views.py

```diff
@@ -1,3 +1,42 @@
-from django.shortcuts import render
+import jwt, requests
 
-# Create your views here.
+from datetime import datetime, timedelta
+
+from django.http  import JsonResponse
+from django.views import View
+
+from bob_morgan.settings import SECRET_KEY, ALGORITHM
+from users.models        import User, SocialNetwork, Platform
+
+class KakaoSigninView(View):
+    def get(self, request):
+        try:
+            kakao_token = request.headers.get('Authorization')
+            headers     = {'Authorization' : f'Bearer {kakao_token}'}
+            KAKAO_URL   = 'https://kapi.kakao.com/v2/user/me'
+
+            response    = requests.get(KAKAO_URL, headers = headers, timeout=3)
+            kakao_user  = response.json()
+
+            if kakao_user.get('code') == -401:
+                return JsonResponse({'message' : 'INVALID_TOKEN'}, status=401)
+
+            kakao_id    = kakao_user['id']
+            email       = kakao_user['kakao_account']['email']
+            name        = kakao_user['kakao_account']['profile']['nickname']
+
+            kakao       = SocialNetwork.objects.get(name='kakao')
+            platform    = Platform.objects.create(platform_id=kakao_id, social_network_id=kakao)
+
+            platform, is_created  = User.objects.get_or_create(
+                platform = platform,
+                defaults = {'email' : email, 'name' : name}
+            )
+
+            access_token = jwt.encode({'id': platform.id}, SECRET_KEY, ALGORITHM)
```

Comment:
```suggestion
            access_token = jwt.encode({'id': platform.id}, settings.SECRET_KEY, settings.ALGORITHM)
```

위의 피드백에 이어 다음과 같이 수정할 수 있습니다.